### PR TITLE
Update go mods and add operator in check

### PR DESCRIPTION
**Pull request by marccampbell**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -53,6 +53,7 @@ require (
 	github.com/russellhaering/goxmldsig v1.1.0 // indirect
 	github.com/segmentio/ksuid v1.0.3
 	github.com/sergi/go-diff v1.1.0
+	github.com/shirou/gopsutil v0.0.0-20190901111213-e4ec7b275ada
```
Comment (laverya) [inline go.mod:56]: was this addition intentional?

**Review by laverya — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (marccampbell) [reply go.mod:56]: I ran `go mod tidy` and it was automatically added. Should we be concerned?

**Review by divolgin — APPROVED** [review]
